lib/whois/parsers/base.rb

```diff
@@ -363,7 +363,7 @@ def response_unavailable?
       protected
 
       def content_for_scanner
-        @content_for_scanner ||= content.to_s.gsub(/\r\n/, "\n")
+        @content_for_scanner ||= "#{content.to_s.gsub(/\r\n/, "\n").gsub(/\n$/, '')}\n"
```

Comment:
As a matter of readability, it's not immidiately clear what we're doing here and why we're doing it.

Reply:
What change would you suggest ?
Would that be acceptable ?

```suggestion
        @content_for_scanner ||= begin
            content_for_scanner = content.to_s
            content_for_scanner.gsub!(/\r\n/, "\n")
            
            # We need a new line at the end, which exists in some cases, and does
            # not in others, we will thus remove the ending new line character if it
            # exists (or do nothing if it does not), then return the content appended
            # with a new line character
            content_for_scanner.gsub!(/\n$/, '')
            "#{content_for_scanner}\n"
        end
```